lib/game_teams_collection.rb

```diff
@@ -331,6 +331,17 @@ def hoa_game_sorter(h_a)
     end
   end
 
+  def head_to_head_hash(teamid)
+    record = Hash.new {|hash, key| hash[key] = []}
+    game_teams.each do |gameteam|
+      record[gameteam.team_id] << gameteam.result if teamid.to_i == gameteam.team_id
+    end
+    id_to_percentage = record.reduce({}) do |acc, (key, value)|
+      acc[key] = (value.count("WIN") / value.length.to_f)
+      acc
+    end
+  end
+
```

Comment:
We have the same method in games collection we should look at both and see whats best